grammar.js

```diff
@@ -177,29 +177,31 @@ module.exports = grammar({
     // Intended for table column names per `:help help-writing`.
     // TODO: children should be $.word (plaintext), not $.atom.
     column_heading: ($) => seq(
-      field('name', seq(choice($._atom_noli, $._uppercase_words), repeat($._atom))),
+      alias(seq(choice($._atom_noli, $._uppercase_words), repeat($._atom)), $.heading),
       '~',
       token.immediate('\n'),
     ),
 
     h1: ($) =>
-      seq(
-        token.immediate(field('delimiter', /============+[\t ]*\n/)),
-        repeat1($._atom),
+      prec(1, seq(
+        alias(token.immediate(field('delimiter', /============+[\t ]*\n/)), $.separator),
+        alias(repeat1($._atom), $.heading),
+        optional(seq($.tag, repeat($._atom))),
```

Comment:
everything before the first tag is `heading` ? should we name it `text` and use that as the pseudo-convention for exposing the "text content" of complex captures? I was experimenting with this but only used it for "fields" (see e.g. `url`).

Reply:
Depends. If we use a common node (not name!) for `text` nodes (that aggregate `word`s?), then we need a field again to distinguish them.

Reply:
And yes, I tried to be consistent across headings (even column headings, which as usual is a massive headache).